catalog/app/website/components/Illustration.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { Box } from '@material-ui/core'
-import { styled } from '@material-ui/styles'
+import { styled } from '@material-ui/core/styles'
```

Comment:
these two are not strictly identical, tho probably it should be okay for our use case